spec/support/methods/feature_test_helper_methods.rb

```diff
@@ -0,0 +1,45 @@
+module FeatureTestHelperMethods
+  def start(the_flow:, at:)
+    visit "/#{at}"
+    ensure_page_has(header: the_flow)
+    click_govuk_start_button
+  end
+
+  def answer(question:, of_type:, with:)
+    ensure_page_has(header: question)
+    case of_type
+    when :radio_button
+      choose with
+    when :check_box
+      check with
+    when :check_boxes
+      raise "For the `check_boxes` type, `with` should be an array" unless with.is_a?(Array)
+
+      with.each do |answer|
+        check answer
+      end
+    when :input_text
+      fill_in "response", with: with
+    when :date
+      raise "For the `date` type, `with` should be a hash" unless with.is_a?(Hash)
```

Comment:
I wonder if being about to pass in a date string more convenient than a hash? Whilst a has is more explicit, feels like a lot more typing.

i.e. `{day: 31, month: 12, year: 2020}` => `"31-12-2020"`

etc... `date = Date.parse("31-12-2020"); date.month;`

Reply:
Yup, makes sense and different date formats and validation better.  Good spot - thanks.